src/report.rs

```diff
@@ -213,12 +213,8 @@ mod test {
     #[test]
     fn add_report_no_dir() {
```

Comment:
I rewrote this test to be true to its name (previously it was just a repeat of the test above). I realize I should mayb pull this out and put in its own PR?